src/pages/code-of-conduct.js

```diff
@@ -0,0 +1,159 @@
+                    Ongoing, unwelcome advances - Fenders is a professional
+                    event, please keep things professional.
+                </li>
+                <li>
+                    Advocating for, or encouraging, any of the above behaviour
+                </li>
+            </ul>
+            <h2>The Slack Group</h2>
+            <p>
+                The Fenders slack group is open to all members of the Perth web
+                community, and we encourage discussion of topics outside of the
+                traditional “front end space”. Please remember that Fenders is a
+                professional group for web developers and designers, as a result
+                if any of the Fenders organisers directly request you stop
+                discussing a topic such as something that promotes hate, abuse
+                or any form of harassment you will be expected to follow their
+                request. As moderators Fenders organisers reserve the right to
+                moderate the group to ensure a safe and welcoming environment
+                for everyone.
+            </p>
+            <p>
+                If you choose to ignore the requests of the Fenders organisers
+                or moderators you may be banned from the slack group and future
+                events.
+            </p>
+            <p>
+                We do not allow unsolicited DMs in the slack group. Please
+                refrain from contacting people directly unless they have
+                indicated that you are fine to do so.
+            </p>
+            <h2>Reporting harassment</h2>
+            <p>
+                If someone makes you feel unsafe or unwelcome, if you notice
+                someone else being harassed or have any concerns please let the
+                Fenders organisers know as soon as possible. We understand that
+                sometimes you might still be concerned, afraid or feel
+                uncomfortable reporting incidents directly so you can also make
+                an anonymous report using the google form (
+                <a href="https://bit.ly/2smU48D)">https://bit.ly/2smU48D)</a>.
+                We will investigate it and take whatever action is necessary to
+                prevent the incident occurring again.
+            </p>
+            <h2>Contact us:</h2>
+            <ul>
+                <li>
+                    Meetup.com (
+                    <a href="https://www.meetup.com/front-end-web-developers-perth/">
+                        www.meetup.com/front-end-web-developers-perth/
+                    </a>
+                    ),
+                </li>
+                <li>
+                    Twitter Direct Message (
+                    <a href="https://twitter.com/fendersperth">
+                        https://twitter.com/fendersperth
+                    </a>
+                    )
+                </li>
+                <li>DM @mandy via the slack group</li>
```

Comment:
Should we include all the organisers?

Reply:
I'd prefer we left it as just me for now please